### ISSUE TITLE
Searching based on concepts referenced via CredentialAlignmentObject

### ISSUE THREAD

**Issue by siuc-nate** [CredentialEngine/CredentialRegistry#570]
The following queries yield 0 results:
```
{
	"ceterms:credentialStatusType": {
		"ceterms:targetNode": "https://credreg.net/ctdl/vocabs/credentialStat/Active"
	}
}
```
```
{
	"ceterms:credentialStatusType": {
		"ceterms:targetNode": "credentialStat:Active"
	}
}
```

But these both work:
```
{
	"ceterms:credentialStatusType": "credentialStat:Active"
}
```
```
{
	"ceterms:credentialStatusType": "credentialStat:Deprecated"
}
```

Also this one does not:
```
{
	"ceterms:credentialStatusType": "https://credreg.net/ctdl/vocabs/credentialStat/Active"
}
```

Was something done to abstract out the Credential Alignment Object, at least in some cases, sort of like how language maps are handled?  That could be useful, if so - but the first two queries should still work, as they follow the structure of the underlying CTDL.

Example credential:
https://credentialengineregistry.org/resources/ce-91d5eb73-f136-4510-91ad-1d4290307240

**Comment by siuc-nate** [comment]
I welcome comments from the Registry team on https://github.com/CredentialEngine/Schema-Development/issues/839 as well.

**Comment by excelsior** [comment]
@siuc-nate There is a bug with how such subresources get indexed. I'm working on a fix.

**Comment by excelsior** [comment]
The bug was caused by the indexer expecting reference properties to have array values. If a reference was a single object, it would be processed as an array of its key-value pairs, leading to unqueryable data in the index.

I fixed it and will reindex all of the affected resources on prod and sandbox. I'll let you know once it's done.

Also, the CredentialAlignmentObject values have not been abstacted yet. Hence, if such an object's target node has a `credentialStat:Active` value, then the same value needs to be used in the query.
